src/views/virtualmachines/details/components/details/components/BootOrderSummary/BootOrderSummary.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+
+import { getDisks, getInterfaces } from '../../../../../utils/selectors';
+import { transformDevices } from '../../utils/bootOrderHelper';
+import BootableDevicesList from '../BootableDevicesList/BootableDevicesList';
+
+import EmptyBootOrderSummary from './EmptyBootSourceSummary';
+
+type BootOrderSummaryProps = {
+  vm: V1VirtualMachine;
+};
+
+const BootOrderSummary: React.FC<BootOrderSummaryProps> = ({ vm }) => {
+  const disks = getDisks(vm);
+  const interfaces = getInterfaces(vm);
+
+  const transformedDevices = transformDevices(disks, interfaces)?.sort(
+    (a, b) => a?.value?.bootOrder - b?.value?.bootOrder,
+  );
+
+  const filtered = [...transformedDevices]?.filter((device) => device?.value?.bootOrder);
+
+  const isNoDevices = filtered?.length === 0;
```

Comment:
This doesn't read correctly because 'is' is used for singular. I would just name is `noDevices`.

Reply:
make sense, changed :)